client/src/app/services/auth.service.ts

```diff
@@ -56,7 +57,7 @@ export class AuthService {
 	}
 
 	// get the token by credentials
-	login(credentials): Promise<any> {
+	login(credentials: Credentials): Promise<any> {

```

Comment:
Nejsem TS odborník, tak bych potřeboval vysvětlit, jak přidáním typu můžeme něco opravit. Jsem myslel, že za běhu tam ty typy pak zase nejsou, takže tohle nemění žádné chování. Nebo ano?

Reply:
Upravil jsem komentář a jméno PR aby to neznělo tak dramaticky :).